Detectors/GlobalTrackingWorkflow/tofworkflow/src/RecoWorkflowSpec.cxx

```diff
@@ -79,11 +79,12 @@ class TOFDPLRecoWorkflowTask
       *tpclab.get() = std::move(*tpclabel);
     }
 
-    auto recPointsPtr = std::make_shared<std::vector<o2::ft0::RecPoints>>();
+    // TODO: can use this simplified line when InputRecord provides the ReturnType helper
+    //InputRecord::ReturnType<std::vector<o2::ft0::RecPoints>> recPoints;
+    decltype(std::declval<InputRecord>().get<std::vector<o2::ft0::RecPoints>>(DataRef{nullptr, nullptr, nullptr})) recPoints;
```

Comment:
Why not getting here ``pc.inputs().get<gs::span<o2::ft0::RecPoints>>("fitrecpoints"))`` directly ?

Reply:
this we can do, I though the object is still ROOT serialized, and then it can not be extracted as span.

Reply:
Now it is!

Reply:
good, then I will change the code, thanks.

Reply:
The object is now retrieved as span.